assets/translate.js

```diff
@@ -94,12 +60,28 @@ $( function () {
 		} );
 	}
 
-	// Change field labels when the source lang changes.
+	// Change the target lang value.
+	if ( $targetLangWidget.length === 1 ) {
+		targetLangWidget = OO.ui.infuse( $targetLangWidget[ 0 ] );
+		targetLangWidget.on( 'change', function () {
+			model.setTargetLang( targetLangWidget.getValue() );
+			// Also modify the form value that will be submitted.
+			$( "input[name='target-lang']" ).val( model.getTargetLang() );
+			if ( $.uls.data.languages[ model.getTargetLang() ] ) {
```

Comment:
Is there a case where this statement is false? Do we need to fortify against it? (Would the label not change properly if this is false or something?)

Reply:
Oh yeah. I think we can drop the conditional all together, because the target lang must always exist in ULS because that's how it was selected. This is I think a remnant from when we were maybe going to allow passing source and target langs in the URL (but even if we do that we should be validating the lang before we get to this point).

I've added a comment to the code.